sidebars.ts

```diff
@@ -1908,6 +1908,7 @@ integrations: [
         collapsed: true,
         link: {type: 'doc', id: 'integrations/google/index'},
         items: [
+          'integrations/google/cloud-bigtable',
```

Comment:
Move `cloud-bigtable` between `cloud-audit` and 'cloud-firewall` so that the it is in alphabetical order.